reproductions/basic-sqlite/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Prisma development playground",
   "main": "index.ts",
   "scripts": {
+    "prisma": "node -e 'console.log(require.resolve(\"@prisma/cli\"))'",
```

Comment:
This seems like a hacky fix, why change the import path?

Reply:
I agree.